src/app/models/api-models/user-gender/user-gender.ts

```diff
@@ -0,0 +1,53 @@
+import {Language} from '../language/language';
+import {LanguageFactory} from '../language/language';
+
+// API attribute interfaces
+interface UserGenderApiAttributes {
+  id: string;
+  name: string;
+  language: Language; // TODO missing relation in docs?
+  languageId: string; // TODO missing in docs?
```

Comment:
Ah, this is a good opportunity to discuss this :) 

These resources are persisted in the database and these strings are loaded from a YAML-file. So they're kinda guaranteed to be translated :) however I can really see a quite strong case for having translations be uniform across all of our API-resources.